doc/event-alarm-framework/event-alarm-framework.md

```diff
@@ -0,0 +1,1265 @@
+Id           Action          Severity   Name                           Timestamp                   Description                                                  
+----------------------------------------------------------------------------------------------------------------------------
+2            RAISE           CRITICAL   DUMMY_ALARM                    2021-05-19T21:39:31.622Z    :- signalHandler: Raising simulated alarm         
+3            CLEAR           CRITICAL   DUMMY_ALARM                    2021-05-19T21:42:34.371Z    :- signalHandler: Clearing simulated alarm        
+4            RAISE           CRITICAL   DUMMY_ALARM                    2021-05-19T21:46:14.371Z    :- signalHandler: Raising simulated alarm         
+5            ACKNOWLEDGE     CRITICAL   DUMMY_ALARM                    2021-05-19T21:48:05.845Z    Alarm id 4 ACKNOWLEDGE.                           
+6            UNACKNOWLEDGE   CRITICAL   DUMMY_ALARM                    2021-05-19T21:53:24.484Z    Alarm id 4 UNACKNOWLEDGE.                         
+7            CLEAR           CRITICAL   DUMMY_ALARM                    2021-05-19T21:55:54.977Z    :- signalHandler: Clearing simulated alarm
+
+sonic# show event recent 24hr 
+----------------------------------------------------------------------------------------------------------------------------
+Id           Action          Severity   Name                           Timestamp                   Description                                                  
+----------------------------------------------------------------------------------------------------------------------------
+2            RAISE           CRITICAL   DUMMY_ALARM                    2021-05-19T21:39:31.622Z    :- signalHandler: Raising simulated alarm         
+3            CLEAR           CRITICAL   DUMMY_ALARM                    2021-05-19T21:42:34.371Z    :- signalHandler: Clearing simulated alarm        
+4            RAISE           CRITICAL   DUMMY_ALARM                    2021-05-19T21:46:14.371Z    :- signalHandler: Raising simulated alarm         
+5            ACKNOWLEDGE     CRITICAL   DUMMY_ALARM                    2021-05-19T21:48:05.845Z    Alarm id 4 ACKNOWLEDGE.                           
+6            UNACKNOWLEDGE   CRITICAL   DUMMY_ALARM                    2021-05-19T21:53:24.484Z    Alarm id 4 UNACKNOWLEDGE.                         
+7            CLEAR           CRITICAL   DUMMY_ALARM                    2021-05-19T21:55:54.977Z    :- signalHandler: Clearing simulated alarm
+
+sonic# show event id 2
+----------------------------------------------
+Event Details - 2 
+----------------------------------------------
+Id:                  2 
+Action:              RAISE 
+Severity:            CRITICAL 
+Type:                DUMMY_ALARM 
+Timestamp            2021-05-19T21:39:31.622Z 
+Description:         :- signalHandler: Raising simulated alarm 
+Source:              simulation 
+
+sonic# show event from 2 to 5
```

Comment:
What happens if the request for entries is not available? what should be the CLI command behaviour?
For example 2 to 5 but only 3?
Is there any check for legal range?

Reply:
If the requested info is not there, no o/p is shown just like any other command. 
The range is 1 - 40,000